### DISCUSSION TITLE
キースイッチがトッププレートにはまりきらず浮いてしまう箇所がある

### DISCUSSION THREAD

**Discussion by wrinkle-ridge** [Yowkees/keyball#643] [Q&A]
# 状況

キースイッチをはめた際、トラックボールがある側のトッププレート穴においてスイッチがはまりきらず浮いてしまいます。
トラックボールのない側のトッププレートでは正常に奥まで差し込めているため、トッププレートの製造不良と思われます。
- スイッチが浮いているプレート
![image](https://github.com/user-attachments/assets/01985338-549d-44c6-9df2-4e29f7a20b40)

- スイッチがはまっているプレート
![image](https://github.com/user-attachments/assets/3961c97c-dbe0-4047-a0f0-b8d9fe9f9dd4)

### 使用しているスイッチ
- Gateron Oil King

# 要望
購入側の過失が原因でなければ、交換等のご対応をお願いできますでしょうか。
お手数おかけしますが、ご確認いただけますと幸いです。

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

ドリル加工が甘い基板は目視検査で除外していますが、間違って混入してしまったのかも知れません。
ご迷惑をお掛けして申し訳ありません。

新しいトッププレートを送付させていただきます。
お手数ですが、発送先等をご相談したいので、下記の白銀ラボウェブショップよりメッセージをいただけますでしょうか。
https://shirogane-lab.net/inquiry

お手数ですがよろしくお願いいたします。

**Reply by wrinkle-ridge** [reply]
お返事ありがとうございます。
フォームより発送先についてご連絡させて頂きました。
ご確認いただけますと幸いです。
